### PR TITLE
Add check for virtlogd

### PR DESCRIPTION
**Pull request by cliping**
Add test for:
1. Check virtlogd after reloading virtlogd.
2. Check virtlogd after restarting libvirtd.

Signed-off-by: Liping Cheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

# avocado run --vt-type libvirt conf_file.qemu_conf.set_virtlogd
JOB ID     : 42da78b74d6783b5fa160517d442029c6a5c4df7
JOB LOG    : /root/avocado/job-results/job-2019-01-04T03.07-42da78b/job.log
 (1/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.default: PASS (113.26 s)
 (2/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.enable_logd: PASS (115.04 s)
 (3/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.reload: PASS (114.14 s)
 (4/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.restart: PASS (144.37 s)
 (5/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.negative_test.invalid: PASS (65.53 s)
RESULTS    : PASS 5 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 585.17 s

File: libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py [hunk 1]
```diff
@@ -135,9 +135,31 @@ def check_pipe_closed(pipe_node):
...
+        """
+        Check service status and return service PID
+        """
+        cmd = ("systemctl status %s | grep 'Active: active'" % service_name)
+        ret = process.run(cmd, shell=True)
+        if ret.exit_status:
+            if service_start == "yes":
+                ret = process.run("systemctl start %s" % service_name, shell=True)
+                if ret.exit_status:
+                    test.fail("%s start failed: %s" % (service_name, ret.stdout))
```
Comment (chunfuwen) [inline libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py]: please take care ret.stdout compatability with python 3

File: libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py [hunk 2]
```diff
@@ -135,9 +135,31 @@ def check_pipe_closed(pipe_node):
         if not process.run(cmd, timeout=90, ignore_status=True, shell=True).exit_status:
             test.fail("pipe node: %s is not closed in qemu gracefully." % pipe_node)
 
+    def check_service_status(service_name, service_start=None):
+        """
```
Comment (chunfuwen) [inline libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py]: miss doc string

File: libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py [hunk 3]
```diff
@@ -135,9 +135,38 @@ def check_pipe_closed(pipe_node):
...
+        ret = process.run(cmd, shell=True)
+        if ret.exit_status:
+            test.fail("Get %s status failed." % service_name)
+        return ret.stdout_text.split()[2]
+
     vm_name = params.get("main_vm", "avocado-vt-vm1")
     expected_result = params.get("expected_result", "virtlogd_disabled")
     stdio_handler = params.get("stdio_handler", "not_set")
+    start_vm = "yes" == params.get("start_vm", "yes")
+    reload_virtlogd = params.get("reload_virtlogd", "no")
```
Comment (chloerh) [inline libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py]: reload_virtlogd = 'yes' == params.get("reload_virtlogd", "no")
then the variable "reload_virtlogd" is bool type, same with restart_libvirtd

**Pull request merged**